utt-replica/signature-processor/src/sigProcessor.cpp

```diff
@@ -88,7 +88,7 @@ SigProcessor::SigProcessor(uint16_t repId,
       timer_handler_timeout_{timer_handler_timeout} {
   msg_handlers->registerMsgHandler(
       bftEngine::impl::MsgCode::Reserved, [&](std::unique_ptr<bftEngine::impl::MessageBase> message) {
-        auto msg = std::make_unique<messages::PartialSigMsg>(message.release());
+        std::unique_ptr<messages::PartialSigMsg> msg((messages::PartialSigMsg*)message.release());
```

Comment:
`reinterpret_cast<messages::PartialSigMsg*>(message.release())`
same for line 297

Reply:
OK